include/stdc/complex.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * (C) Copyright 2002-2015 Diomidis Spinellis
+ *
+ * This file is part of CScout.
+ *
+ * CScout is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * CScout is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with CScout.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/*
+ *	ISO C99:  7.3 Complex arithmetic	<complex.h>
+ */
+
+#ifndef _COMPLEX_H
+#define _COMPLEX_H   1
+#define complex _Complex
+#define imaginary _Imaginary
+
+#define _Complex_I const float _Complex
+#define _Imaginary_I const float _Imaginary
+
+#define I _Imaginary_I
+
```

Comment:
You're missing the function declarations [listed in the standard](http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/complex.h.html).